spec/requests/user_request_spec.rb

```diff
@@ -15,8 +15,10 @@
       create(:account, idme_uuid: mhv_user.uuid)
       sign_in_as(mhv_user)
       allow_any_instance_of(User).to receive(:edipi).and_return(edipi)
-      VCR.use_cassette('va_profile/veteran_status/va_profile_veteran_status_200', allow_playback_repeats: true) do
-        get v0_user_url, params: nil, headers: v0_user_request_headers
+      VCR.use_cassette('va_profile/demographics/demographics', allow_playback_repeats: true) do
+        VCR.use_cassette('va_profile/veteran_status/va_profile_veteran_status_200', allow_playback_repeats: true) do
+          get v0_user_url, params: nil, headers: v0_user_request_headers
+        end
       end
```

Comment:
Instead of nesting calls to `VCR.use_cassette`, I would suggest the following:

```
    cassettes = [
      { name: 'profile/demographics' },
      { name: 'va_profile/veteran_status/va_profile_veteran_status_200' }
    ]
    VCR.use_cassettes cassettes do
      get v0_user_url, params: nil, headers: v0_user_request_headers
    end
```

An alternate way, utilizing the rspec `around` block could be:

```
 around do |test_case|
    cassettes = [
      { name: 'profile/demographics', allow_playback_repeats: true },
      { name: 'va_profile/veteran_status/va_profile_veteran_status_200', match_requests_on: %i[method body], allow_playback_repeats: true }
    ]
    VCR.use_cassettes cassettes do
      test_case.run
    end
  end
```

Note that you can add configuration for the cassette within the hash for that cassette. Typically you would only want to use `allow_playback_repeats: true` if the test case will make more than one request for the same resource.

Reply:
1. Interesting that I could not find any documentation anywhere on how to do this. I tried it out, but could not get it working. I found info that says the preference is to use the blocks anyways and if not use the insert/eject method, but that method is discouraged as it can lead to bad states in tests.
2. As far as the `allow_playback_repeats`, I wanted to match what was on the other cassette. I was able to remove some of it, but some tests do make multiple requests because they have multiple tests in one test.